gitlint/__init__.py

```diff
@@ -217,49 +233,52 @@ def main(argv, stdout=sys.stdout, stderr=sys.stderr):
     gitlint_config = get_config(repository_root)
     json_result = {}
 
-    for filename in sorted(modified_files.keys()):
-        rel_filename = os.path.relpath(filename)
-        if not json_output:
-            stdout.write('Linting file: %s%s' %
-                         (termcolor.colored(rel_filename, attrs=('bold',)),
-                          linesep))
-        if arguments['--force']:
-            modified_lines = None
-        else:
-            modified_lines = vcs.modified_lines(filename,
-                                                modified_files[filename],
-                                                commit=commit)
-
-        result = linters.lint(
-            filename, modified_lines, gitlint_config)
-        result = result[filename]
-
-        output_lines = []
-        if result.get('error'):
-            output_lines.extend(
-                '%s: %s' % (ERROR, reason) for reason in result.get('error')
-            )
-            linter_not_found = True
-        if result.get('skipped'):
-            output_lines.extend(
-                '%s: %s' % (SKIPPED, reason) for reason in result.get('skipped')
-            )
-        if result.get('comments', []) == []:
-            if not output_lines:
-                output_lines.append(OK)
-        else:
-            files_with_problems += 1
-            for data in result['comments']:
-                formatted_message = format_comment(data)
-                output_lines.append(formatted_message)
-                data['formatted_message'] = formatted_message
-
-        if json_output:
-            json_result[filename] = result
-        else:
-            output = linesep.join(output_lines)
-            stdout.write(output)
-            stdout.write(linesep + linesep)
+    args = [modified_files,
+            commit,
+            arguments['--force'],
+            gitlint_config]
+
+    with futures.ThreadPoolExecutor(max_workers=5) as executor:
+        future_to_file = {executor.submit(processFile, file, *args):
+                          file for file in sorted(modified_files.keys())}
```

Comment:
Explicitly set the arguments here instead of using *args

Reply:
Changed.